src/components/Atoms/AddTaskButton/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+import TEXT from "../../../variables/texts";
+import COLOR from "../../../variables/color";
+import plus from "../../../assets/svg/plus.svg";
+
+const AddTaskButton = ({ onClick }) => {
+  return (
+    <StyledTaskButton onClick={onClick}>
+      <StyledGreenOval></StyledGreenOval>
+      <StyledTask>
+        <img src={plus}></img>
+        <StyledAddTask>タスクを追加</StyledAddTask>
+      </StyledTask>
+    </StyledTaskButton>
+  );
+};
+
+export default AddTaskButton;
+
+const StyledGreenOval = styled.div`
+  background-color: ${COLOR.GREEN_TRANSPARENT};
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 100%;
+  border-radius: 12px;
+  transition: all 0.2s ease;
+`;
+
+const StyledAddTask = styled.div`
+  color: ${COLOR.GREEN};
+  ${TEXT.S};
+  margin-left: 10px;
+  white-space: nowrap;
+`;
+
+const StyledTask = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 2px 6px;
+`;
+
+const StyledTaskButton = styled.button`
+  border: none;
+  outline: none;
+  background: none;
+  padding: 0px;
+  position: relative;
+  cursor: pointer;
+  & > img {
+    width: 20px;
+    height: 20px;
+  }

```

Comment:
```suggestion
  & img {
    width: 20px;
    height: 20px;
  }
```
小結合子 `>` は直接の子要素しか対象としません。
`AddTaskButton` と `img` は孫の関係なので、子結合子を使う場合スタイルが当たりません。
子孫結合子（半角スペース）を使うのが手っ取り早くてよきかと思います。

セレクタについて：https://www.notion.so/shinonome-inc/CSS-cafffc6d680243c5bfa304847092b21d?pvs=4